src/index.js

```diff
@@ -1,6 +1,21 @@
 // Polyfill `Promise`.
 window.Promise = window.Promise || require('promise-polyfill');
 
+// Check before the polyfill runs
+window.hasNativeWebVRImplementation = !!navigator.getVRDisplays;
```

Comment:
Implementation conveys better the meaning than support. Browsers ship or not a native implementation of the APIs. Support has an ambiguous meaning in this context.